src/EditorFeatures/CSharpTest/Completion/CompletionProviders/Snippets/CSharpForEachSnippetCompletionProviderTests.cs

```diff
@@ -225,7 +225,6 @@ public async Task InsertForEachSnippetInParenthesizedLambdaExpressionRegularTest
     {
         $$
     }
-
```

Comment:
For cases where the last token is a closing curly brace, should we special case and keep the elastic trivia?